Formula/cava.rb

```diff
@@ -0,0 +1,60 @@
+class Cava < Formula
+  desc "Console-based Audio Visualizer for ALSA"
+  homepage "https://github.com/karlstav/cava"
+  url "https://github.com/karlstav/cava/archive/0.7.4.tar.gz"
+  sha256 "fefd3cc04d41b03ca416630cafadbfda6c75e2ca0869da1f03963dcb13e1ecb7"
+  license "MIT"
+  head "https://github.com/karlstav/cava.git", branch: "master"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool"  => :build
+  depends_on "fftw"
+  depends_on "iniparser"
+  depends_on "portaudio"
+
+  uses_from_macos "ncurses"
```

Comment:
Didn't you say this doesn't link with macOS `ncurses`? We can use Homebrew `ncurses` and ignore the audit failure.

Reply:
It links with macos ncurses as long as the inreplace takes place. For avoiding the inreplace in the future the devs already pushed to master. But version 0.7.4 still uses this inreplace. I made this inreplace conditional.

Reply:
Ok, I see. If it works fine with macOS ncurses then we can leave it.